chingu-voyages/v41-bears-team-20
# Display profile with data from backend

Closes #93 

**HOW TO TEST**
- Setup:
  - clone pull request `git fetch origin refs/pull/83/head:pr-83`
  - switch to branch: `git checkout pr-83`
  - Run `npm i`
  - Run `cd server`
  - Run `npm run db:docker:up`
  - Run `docker ps` to confirm containers are running
  - Run `npx prisma migrate` or `npx prisma reset`
  - Run `npm run start:dev`
  - Open new terminal instance
  - Run `cd server`
  - Run `npx prisma studio` --> opens up database viewer
    - if there is no data present, then seed the database. RUN: `npx prisma db seed`
  - Open new terminal instance
  - Run `cd client`
  - Run `npm run dev`
  
- Test:
  - Go to `server/src/prisma/data.ts`
  - Copy a student's email & password and tutor's email & password
  - Login with student credentials
  - Navigate to `http://localhost:5173/account`
  - Verify you see the logged in user's profile
  - Logout
    - you may see the signout button in navbar, in that case go to `http://localhost:5173/account/login`
  - Login with tutor credentials
  - Navigate to `/account`
  - Verify you see the logged in user's profile
  - Logout
  - Go to `http://localhost:5173/account/:id` where `:id` is any number between 1-10
  - Verify you see valid profiles
